packages/strapi-admin/test/admin-auth.test.e2e.js

```diff
@@ -0,0 +1,168 @@
+// Helpers.
+const { registerAndLogin } = require('../../../test/helpers/auth');
+const { createAuthRequest } = require('../../../test/helpers/request');
+
+let rq;
+
+expect.extend({
+  stringOrNull(received) {
+    const pass = typeof received === 'string' || received === null;
+    if (pass) {
+      return {
+        message: () => `expected ${received} not to be null or a string`,
+        pass: true,
+      };
+    } else {
+      return {
+        message: () => `expected ${received} to be null or a string`,
+        pass: false,
+      };
+    }
+  },
+});
+
+describe('Content Manager End to End', () => {
+  beforeAll(async () => {
+    const token = await registerAndLogin();
+    rq = createAuthRequest(token);
+  }, 60000);
+
+  describe('Login', () => {
+    test('Can connect successfuklly', async () => {
+      const res = await rq({
+        url: '/admin/login',
+        method: 'POST',
+        body: {
+          email: 'admin@strapi.io',
+          password: 'pcw123',
+        },
+      });
+
+      expect(res.statusCode).toBe(200);
+      expect(res.body.data).toMatchObject({
+        token: expect.any(String),
+        user: {
+          firstname: expect.stringOrNull(),
+          lastname: expect.stringOrNull(),
+          username: expect.stringOrNull(),
+          email: expect.any(String),
+          isActive: expect.any(Boolean),
+        },
```

Comment:
What about the roles[]?

Reply:
Will add this when we add the default roles so we check the user has a role.